airbyte-config/persistence/src/test/java/io/airbyte/config/persistence/ConfigPersistenceBuilderTest.java

```diff
@@ -103,22 +104,22 @@ void tearDown() throws Exception {
 
   @Test
   public void testCreateDbPersistenceWithYamlSeed() throws IOException {
-    ConfigPersistence dbPersistence = ConfigPersistenceFactory.build(configs).setupDatabase(true).get().createDbPersistenceWithYamlSeed();
+    ConfigPersistence dbPersistence = new ConfigPersistenceFactory(configs, true, false).createDbPersistenceWithYamlSeed();
```

Comment:
I'd like to keep constructing the factory using the builder. In this way, we include the builder in the unit test, and if in the future more parameters are added to the factory, it does not always require us to manually change every test case.